pygelf/handlers.py

```diff
@@ -159,15 +163,21 @@ def __init__(self, host, port, compress=True, path='/gelf', timeout=5, **kwargs)
         if compress:
             self.headers['Content-Encoding'] = 'gzip,deflate'
 
+        if username and password:
+            self.headers['Authorization'] = get_basic_auth_header(username, password)
+
     def emit(self, record):
         data = self.convert_record_to_gelf(record)
         connection = httplib.HTTPConnection(host=self.host, port=self.port, timeout=self.timeout)
         connection.request('POST', self.path, data, self.headers)
+        res = connection.getresponse()
+        if res.status != httplib.ACCEPTED:
+            raise Exception(f'GELF reply is: {res.status} {res.reason}')
```

Comment:
Why do we need to wait for a response here?

Also, I don't think we should actually throw an exception - this case should be covered by tests in a project that uses `pygelf`. The logging library shouldn't be responsible for handling exceptions like this.